packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1281,6 +1281,82 @@ void main() {
     await checkDragHandleAndColors();
   });
 
+  testWidgets('Drag handle interactive area size at minimum possible size', (WidgetTester tester) async {
+
```

Comment:
Nit: Extra newline here makes it feel like the `testwidgets` line above and the `buildScaffold` line below are decoupled. Might help test readability to remove it.